checkov/terraform/checks/resource/gcp/GoogleKMSRotationPeriod.py

```diff
@@ -0,0 +1,24 @@
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+
+NINETY_DAYS = {'d': 90, 'h': 2160, 'm': 129600, 's': 7776000}
+ONE_DAY = {'d': 1, 'h': 24, 'm': 1440, 's': 86400}
```

Comment:
@arielkru According to terraform, [rotation_period](https://www.terraform.io/docs/providers/google/r/kms_crypto_key.html) will always be in seconds. Why the hassle?

Reply:
I've done it based on the the benchmark (which refers to gcloud), your'e right.